openedx/core/djangoapps/programs/signals.py

```diff
@@ -34,12 +34,6 @@ class of the object instance that sent this signal
         None
 
     """
-    # Import here instead of top of file since this module gets imported before
-    # the programs app is loaded, resulting in a Django deprecation warning.
-    from openedx.core.djangoapps.programs.models import ProgramsApiConfig
```

Comment:
@MatthewPiatetsky I agree with removing use of this particular config model. However, instead of scheduling the task only to have it given up on if `is_learner_issuance_enabled` is `False`, I wonder if we should be checking `is_learner_issuance_enabled` here, too. That way, if we need to disable certification, existing tasks aren't completed, and new tasks aren't scheduled.